frontend2/src/views/Tournaments.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useContext, useState, useEffect, useRef } from "react";
+import { EpisodeContext } from "../contexts/EpisodeContext";
+import ReactMarkdown from "react-markdown";
+import { NavLink, Link } from "react-router-dom";
+import { BC23_TOURNAMENTS, TourneyPage } from "../content/bc23";
+import { type Tournament } from "../utils/types";
+import { getAllTournaments } from "../utils/api/episode";
+import BattlecodeTable from "../components/BattlecodeTable";
+// import { CurrentUserContext } from "../contexts/CurrentUserContext";
+import Icon from "../components/elements/Icon";
+
+const isInternalLink = (to: string): boolean => {
+  const url = new URL(to, window.location.origin);
+  return url.hostname === window.location.hostname;
+};
+
+const MarkdownSection: React.FC<{ children: string; className?: string }> = ({
```

Comment:
Refactor to component called `Markdown.tsx` and make sure to accept className prop!